osCam/userconfig/views.py

```diff
@@ -41,7 +43,14 @@ def configPage(request):
                     usermp.append({"username": name, "user_id":name_id, "IP ADDRESS":ip_addr})
 
         content = Json.dumps(usermp)
-    return JsonResponse({'response': 'SUCCESS', 'type':'GET', "Client-IP::LOOKUP":content, 'PAGE': page, 'data':response.get('response')}, safe=True)
+    #return JsonResponse({'response': 'SUCCESS', 'type':'GET', "Client-IP::LOOKUP":content, 'PAGE': page, 'data':response.get('response')}, safe=True)
+
+    # if there are no cameras create a dummy camera
+    # if page_data['cameras'].exists():
+    #    page_data['cameras'][0] = 'camera 1'
+
+
```

Comment:
If you like to save your work and or save where you left off for when you come back, what I like to do is push **DRAFT**-PR up to github then I clean up my code and remove all unnecessary comments. - BUT - since you did a draft-PR, you have just saved a version of this & can check it out "checkout" anytime. Furhtermore, you could also just leave a "draft" version on your local repo. I just thought you may appreciate the DRAFT-PR feature of Git as I saw this =)

Reply:
never heard of the draft feature, was just going to use another branch. 

all these comments were things either broken or otherwise slowing me down but did not have time to also work on